src/deploy.js

```diff
@@ -0,0 +1,23 @@
+require('./setup');
+
+const ether = require('./ether');
+const send = require('./send');
+
+const { ERC1820_REGISTRY_DEPLOY_TX } = require('./data');
+
+async function ERC1820Registry (from) {
```

Comment:
I think we should check, before deploying, if the contract already exists at the target address. This will allow us to report the error much more nicely.

Reply:
This makes sense, yes.